package.json

```diff
@@ -67,7 +67,16 @@
     "blind-threshold-bls": "npm:@celo/blind-threshold-bls@1.0.0-beta",
     "ganache": "npm:@celo/ganache@7.8.0-unofficial.0",
     "@types/bn.js": "4.11.6",
-    "bignumber.js": "9.0.0"
+    "bignumber.js": "9.0.0",
+    "crypto-js": "4.2.0",
+    "flat": "5.0.1",
+    "protobufjs": "7.2.4",
+    "@babel/traverse": "7.23.2",
+    "node-fetch": "2.6.7",
```

Comment:
How were these versions determined? 

Seems that they are not the latest version.

Reply:
The versions was determined by CVEs reports (using trivy or docker scout). I've just bumped the packages to versions where `CRITICAL` and `MAJOR` CVEs were fixed, but I've not checked if there were more recent versions for each package.